backend/src/main/java/com/bulletjournal/messaging/MessagingService.java

```diff
@@ -226,6 +236,59 @@ public void sendTaskDueNotificationAndEmailToUsers(List<Task> taskList) {
         return paramsList;
     }
 
+    public void sendAppInvitationEmailsToUser(String inviter, List<String> emails) {
+        LOGGER.info("Sending app invitation emails...");
+        try {
+            List<MailjetEmailParams> emailParamsList = new ArrayList<>();
+            for (String email : new HashSet<>(emails)) {
+                MailjetEmailParams mailjetEmailParams =
+                    createEmailPramsForAppInvitation(inviter, this.getAvatar(inviter), email);
+                if (mailjetEmailParams != null) {
+                    emailParamsList.add(mailjetEmailParams);
+                }
+            }
+            mailjetClient.sendAllEmailAsync(emailParamsList);
+        } catch (Exception e) {
+            LOGGER.error("sendAppInvitationEmailsToUser failed", e);
+        }
+    }
+
+    public MailjetEmailParams createEmailPramsForAppInvitation(String inviter,
+                                                               String inviterAvatar,
+                                                               String email) {
+        if (!this.isValidEmailAddr(email)) {
+          LOGGER.error("Invalid app invitation email address: {}", email);
+          return null;
+        }
+
+        if (inviter == null || inviterAvatar == null) {
+          LOGGER.error("APP Invitation: Invalid inviter infor");
+          return null;
+        }
+
+        String title = inviter + " invited your to join BulletJournal";
+
+        return new MailjetEmailParams(
+            Arrays.asList(new ImmutablePair(null, email)),
+            title,
+            null,
+            Template.APP_INVITATION,
+            APP_URL_PROPERTY,
+            APP_BASIC_URL,
```

Comment:
Use our home page url https://bulletjournal.us/home/index.html
Let the invitees go to our home page